x-pack/test/security_solution_api_integration/test_suites/siem_migrations/utils/rules.ts

```diff
@@ -109,14 +137,50 @@ const assertStatusCode = (statusCode: number, response: SuperTest.Response) => {
   }
 };
 
+export interface GetRuleMigrationParams {
+  /** `id` of the migration to get rules documents for */
+  migrationId: string;
+  /** Optional page number to retrieve */
+  page?: number;
+  /** Optional number of documents per page to retrieve */
+  perPage?: number;
+  /** Optional field of the rule migration object to sort results by */
+  sortField?: string;
+  /** Optional direction to sort results by */
+  sortDirection?: 'asc' | 'desc';
+  /** Optional parameter to filter documents */
+  filters?: RuleMigrationFilters;
+  /** Optional expected status code parameter */
+  expectStatusCode?: number;
+}
+
 export const migrationRulesRouteHelpersFactory = (supertest: SuperTest.Agent) => {
   return {
-    get: async (
-      migrationId: string,
-      expectStatusCode: number = 200
-    ): Promise<{ body: GetRuleMigrationResponse }> => {
+    get: async ({
+      migrationId,
+      page,
+      perPage,
+      sortField,
+      sortDirection,
+      filters,
+      expectStatusCode = 200,
+    }: GetRuleMigrationParams): Promise<{ body: GetRuleMigrationResponse }> => {
       const response = await supertest
         .get(replaceParams(SIEM_RULE_MIGRATION_PATH, { migration_id: migrationId }))
+        .query({
+          page,
+          per_page: perPage,
+          sort_field: sortField,
+          sort_direction: sortDirection,
+          search_term: filters?.searchTerm,
+          ids: filters?.ids,
+          is_prebuilt: filters?.prebuilt,
+          is_installed: filters?.installed,
+          is_fully_translated: filters?.fullyTranslated,
+          is_partially_translated: filters?.partiallyTranslated,
+          is_untranslatable: filters?.untranslatable,
+          is_failed: filters?.failed,
+        })
```

Comment:
NIT: Consider using the real parameter format from the `GetRuleMigrationRequestQuery` type instead of doing this conversion.

Reply:
Sounds great! I will update it in my next PR.